arxiv/main.tex

```diff
@@ -45,15 +45,18 @@
 \begin{abstract}
     This paper presents a new way of selecting an initial solution for the
     \(k\)-modes algorithm that allows for a notion of mathematical fairness and
-    a leverage of the data that the common initialisations from literature do
+    a leverage of the data that the base initialisations from literature do
     not. The method, which utilises the Hospital-Resident Assignment Problem to
-    find the set of initial cluster centroids, is compared with the current
-    initialisations on both benchmark datasets and a body of newly generated
-    artificial datasets. Based on this analysis, the proposed method is shown to
-    outperform the other initialisations in the majority of cases, especially
-    when the number of clusters is optimised. In addition, we find that our
-    method outperforms the leading established method specifically for
-    low-density data.
+    find the set of initial cluster centroids, is compared with the original
+    initialisation for \(k\)-modes as well as the next most popular method. In
+    order to highlight the merits of the proposed method two stages of analysis
+    are presented. The first uses benchmark datasets to give familiarity whilst
+    the second utilises a body of artificial datasets that were generated to
+    reveal the nuances of each method. Based on this analysis, the proposed
+    method is shown to outperform the other initialisations in the majority of
+    cases, especially when the number of clusters is optimised. In addition, the
+    proposed method is found to outperform the popular established method
+    specifically for low-density data.
```

Comment:
I prefer the style and sentences you've got in the introduction, I would specifically mention Huang and Cao in the abstract, and mention the analysis that you conclude the paragraph in the introduction with, I'd be more specific, and there is no harm in repeating sentences between the abstract and introduction. I think I'd also get rid of "mathematical fairness". That's too vague a concept, What about prior knowledge? Something like that?